README.md

```diff
@@ -64,6 +64,8 @@ python manage.py loaddata
 ## Running NYC Councilmatic locally
 
 ``` bash
+python manage.py createcachetable
+python manage.py collectstatic
```

Comment:
don't need collectstatic to run councilmatic locally, if we have debug set to true. i'm thinking we should have part of the setup instructions broken into 2 sections, for running locally vs for deploying